applications/NXxas_new.nxdl.xml

```diff
@@ -0,0 +1,230 @@
+# For further information, see http://www.nexusformat.org
+-->
+<definition
+    name="NXxas_new"
+    extends="NXobject"
+    type="group"
+    category="application"
+    xmlns="http://definition.nexusformat.org/nxdl/3.1"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd "
+    >
+    <symbols>
+        <doc>
+		    The symbol(s) listed here will be used below to coordinate datasets with the same shape.
+        </doc>
+		<symbol name="nEnergy">
+		    <doc>Number of energy data points</doc>
+        </symbol>
+        <symbol name="nTransitions">
+		    <doc>Number of electronic transitions</doc>
+        </symbol>
+	</symbols>
+    <doc>
+      This is an application definition for X-ray absorption spectroscopy. 
+    </doc>
+    <group type="NXentry">
+        <field name="definition">
+            <doc> Official NeXus NXDL schema to which this file conforms. TODO: replace NXxas </doc>
+            <enumeration>
+                <item value="NXxas_new"></item>
+            </enumeration>
+        </field>
+        <field name="mode" type="NX_CHAR">
+            <doc></doc>
+            <enumeration>
+                <item value="transmission">
+                    <doc>
+                        TODO
+                    </doc>
+                </item>
+                <item value="fluorescence yield">
+                    <doc>
+                        TODO total or partial fluorescence yield
+                    </doc>
+                </item>
+                <item value="herfd">
+                    <doc>
+                        TODO high energy resolution fluorescence detected
+                    </doc>
+                </item>
+                <item value="electron yield">
+                    <doc>
+                        TODO total or partial electron yield
+                    </doc>
+                </item>
+                <item value="luminescence">
+                    <doc>
+                        TODO
+                    </doc>
+                </item>
+                <item value="calculated">
+                    <doc>
+                        TODO
+                    </doc>
+                </item>
+                <item value="x-ray Raman">
+                    <doc>
+                        TODO
+                    </doc>
+                </item>
+                <item value="electron energy loss">
+                    <doc>
+                        TODO
+                    </doc>
+                </item>
+                <item value="dafs">
+                    <doc>
+                        TODO
+                    </doc>
+                </item>
+                <item value="other">
+                    <doc>
+                        TODO
+                    </doc>
+                </item>
+            </enumeration>
+        </field>
+        <group type="NXelement" name="element">
+            <doc>Excited element</doc>
```

Comment:
What if I want to do a scan that includes multiple absorption edges (e.g. Carbon K-edge and K L-edge)? Can I specify multiple elements?